src/modules/Builder/pages/BuilderApplet/BuilderApplet.tsx

```diff
@@ -78,12 +77,25 @@ export const BuilderApplet = () => {
     [],
   );
 
+  const { errors } = useFormState({
+    control: methods.control,
```

Comment:
why not to extract all variables from methods?

Reply:
useFormState accepts only control and name (and another 2 flags don't extracted from methods returned from useForm())

Reply:
for example you can do like this: 
const { control, watch, setValue, getValues, register, unregister, clearErrors, reset } = methods;

Reply:
i missed your point, fixed